src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Random random = new Random();
+
+    public Ball getRandomBall() {
+        Ball newBall = new Ball(colorSupplier.getRandomColor(), random.nextInt(100));
```

Comment:
magic numbers should be constants with proper naming
```suggestion
        Ball newBall = new Ball(colorSupplier.getRandomColor(), random.nextInt(MAX_BALL_NUMBER));
```